### PR TITLE
Day5

### PR DESCRIPTION
**Pull request by rameezjhaveri**

### CODE REVIEW

File: src/test/scala/day5/CalculatorTest.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package day5
+import resources.BaseTestingClass
+
+class CalculatorTest extends BaseTestingClass {
```
Comment (tvaidotas) [inline src/test/scala/day5/CalculatorTest.scala:4]: I would like to see more tests, testing negative values, both numbers 0, one number 0, trying to give a larger number than the method can handle, division by zero etc.

File: src/test/scala/resources/BaseTestingClass.scala [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package resources
+import org.scalatest.FunSuite
+
+abstract class BaseTestingClass extends FunSuite {
```
Comment (tvaidotas) [inline src/test/scala/resources/BaseTestingClass.scala]: you don't need {} as there's nothing in the body
